src/plugins/smart/smartmontools.c

```diff
@@ -33,7 +33,7 @@
 #include "smart-private.h"
 
 /**
- * SECTION: smart-smartmontools
+ * SECTION: smart
```

Comment:
This will unfortunately override the docs generated by the libatasmart plugin. This naming was intentional.

Reply:
Ok, I found a different solution for this.